### PR TITLE
i18n(ko-KR): add `supabase.mdx` & `vue.mdx`

### PR DESCRIPTION
**Pull request by yeonjulee1005**
#### Description (required)

- add `supabase.mdx` 
- add `vue.mdx`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by yeonjulee1005** [comment]
help some one... 😢

**Comment by yeonjulee1005** [comment]
@Waxer59 
excuse me, i have a help for you... 
why did'nt pass `vercel-docs` ? 
could you help me?

File: src/content/docs/en/reference/errors/generate-content-types-error.mdx [hunk 1]
```diff
@@ -1,17 +1,8 @@
...
 title: Failed to generate content types.
 i18nReady: true
 githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
 ---
-import DontEditWarning from '~/components/DontEditWarning.astro'
-
-<DontEditWarning />
-
 
 > **GenerateContentTypesError**: `astro sync` command failed to generate content collection types: ERROR_MESSAGE
```
[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/en/reference/errors/generate-content-types-error.mdx:8]: @dreyfus92 
Thank you so much for your reply!
The same error occurred even before I corrected the part you mentioned.
The area you captured was judged to be an error and was deleted!
Let’s roll back again!

[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/en/reference/errors/generate-content-types-error.mdx]: it's not working 😢 😢

[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/en/reference/errors/generate-content-types-error.mdx]: oh!! thank you soooooo much!!! 😢

[the remark this answers is not in the document]
Comment (dreyfus92) [reply src/content/docs/en/reference/errors/generate-content-types-error.mdx]: This problem is related with the `frontmatter` not sure if the korean characters might be messing something when they're parsed. Cause I can't seem to find something out of the ordinary.

[the remark this answers is not in the document]
Comment (dreyfus92) [reply src/content/docs/en/reference/errors/generate-content-types-error.mdx]: ohh, missing quotes 😁

[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/en/reference/errors/generate-content-types-error.mdx]: @dreyfus92  😢  T.T ... 

I missed `<head>` 's quotes  in `description`

Thank you for your helping me!!! ❤️

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+import { defineConfig } from 'astro/config';
+import vue from '@astrojs/vue';
+
+export default defineConfig({
+  // ...
+  integrations: [
+    vue({
+      template: {
+        compilerOptions: {
+          // treat any tag that starts with ion- as custom elements
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:125]: Can you translate this comment?

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+```js title="astro.config.mjs"
+import { defineConfig } from 'astro/config';
+import vue from '@astrojs/vue';
+
+export default defineConfig({
+  // ...
+  integrations: [
+    vue({
+      jsx: {
+        // treat any tag that starts with ion- as custom elements
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:186]: This one as well.

**Review by dreyfus92 — COMMENTED** [review]
As a `non-native speaker` of korean, I can leave just a couple of suggestions. I would appreciate if our amazing translator @jsparkdev could jump in to validate syntax & grammar 😁

File: src/content/docs/ko/guides/integrations-guide/vue.mdx (hunk 2 above)
Comment (yeonjulee1005) [reply src/content/docs/ko/guides/integrations-guide/vue.mdx:125]: ah! I missed annotation section!
I,ll translate it!🙇🏻‍♀️🙇🏻‍♀️

File: src/content/docs/ko/guides/integrations-guide/vue.mdx (hunk 3 above)
Comment (yeonjulee1005) [reply src/content/docs/ko/guides/integrations-guide/vue.mdx]: I solved it:) Thank you!':)

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```sh
+  yarn add vue
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+그런 다음, `integrations` 속성을 사용하여 `astro.config.*` 파일에 통합적용 해줍니다:
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:81]:
```suggestion
그런 다음 `integrations` 속성을 사용하여 `astro.config.*` 파일에 통합을 적용합니다.
```

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+export default defineConfig({
+  // ...
+  integrations: [vue()],
+});
+```
+
+## 시작하기
+
+Astro에 첫 번째 Vue 컴포넌트를 사용하려면,  [UI 프레임워크 문서](/ko/core-concepts/framework-components/#using-framework-components) 로 이동해줍니다. 이 문서에서 아래와 같은 내용을 확인할 수 있습니다:
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:95]:
```suggestion
Astro에서 첫 번째 Vue 컴포넌트를 사용하려면 [UI 프레임워크 문서](/ko/core-concepts/framework-components/#using-framework-components)로 이동하세요. 다음을 확인할 수 있습니다.
```

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+
+* 📦 프레임워크 컴포넌트가 로드되는 방식,
+* 💧 클라이언트 측(client-side) 하이드레이션 옵션 및
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:98]:
```suggestion
* 💧 클라이언트 측 하이드레이션 옵션
```

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+* 🤝 프레임워크를 함께 사용하고 혼합 사용할 수 있는 기회
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:99]:
```suggestion
* 🤝 프레임워크를 함께 혼합하고 중첩할 수 있는 기회
```

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+
+## 문제 해결
+
+도움이 필요할 경우, [Discord](https://astro.build/chat) 의 `#support` 채널을 확인해줍니다. 우리의 친절한 지원팀 멤버들이 도와드립니다!
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:103]:
```suggestion
도움이 필요하면 [Discord](https://astro.build/chat)의 `#support` 채널을 확인하세요. 우리의 친절한 지원팀 멤버들이 도와드리겠습니다!
```

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+
+또한 [Astro 통합 문서](/ko/guides/integrations-guide/) 를 통해, 더 많은 통합과정에 대한 내용을 확인합니다.
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:105]:
```suggestion
통합에 대한 자세한 내용은 [Astro 통합 문서](/ko/guides/integrations-guide/)를 확인하세요.
```

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 10]
```diff
@@ -0,0 +1,192 @@
...
+
+## 기여
+
+이 패키지는 Astro의 코어 팀이 유지관리 합니다. 문제가 PR은 언제든 환영합니다!
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:109]:
```suggestion
이 패키지는 Astro 코어 팀에서 유지 관리합니다. 문제를 보고하거나 PR을 제출하는 것은 언제든 환영합니다!
```

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 11]
```diff
@@ -0,0 +1,192 @@
...
+          isCustomElement: (tag) => tag.startsWith('ion-'),
+        },
+      },
+      // ...
+    }),
+  ],
+});
+```
+
+### 앱 진입점
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:135]:
```suggestion
### appEntrypoint
```

옵션으로 전달되는 객체의 key입니다. 번역이 필요하지 않은 것으로 판단됩니다.

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 12]
```diff
@@ -0,0 +1,192 @@
...
+
+특정 기능 가져오기(예: `appEntrypoint: "/src/pages/_app"`) 를 `appEntrypoint`에 설정하여 Vue `app` 인스터스 설정을 확장할 수 있습니다.
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:137]:
```suggestion
`appEntrypoint` 옵션을 루트 상대 가져오기 지정자로 설정하여 (예: `appEntrypoint: "/src/pages/_app"`) Vue `app` 인스턴스를 확장할 수 있습니다.
```

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 13]
```diff
@@ -0,0 +1,192 @@
...
+
+이 파일의 기본 내보내기는 랜더링 전에 Vue `app` 인스턴스는 [커스텀 Vue 플러그인](https://vuejs.org/guide/reusability/plugins.html), `app.use` 및 기타 사용자 정의된 고급 사용사례와 같은 것을 허용하는 기능이어야 합니다.
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:139]:
```suggestion
이 파일의 기본 내보내기는 렌더링 전에 Vue `App` 인스턴스를 전달받는 함수여야 하며,  [사용자 정의 Vue 플러그인](https://ko.vuejs.org/guide/reusability/plugins.html), `app.use` 및 고급 사용 사례를 위한 기타 사용자 정의를 허용합니다.
```

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 14]
```diff
@@ -0,0 +1,192 @@
...
+import i18nPlugin from 'my-vue-i18n-plugin';
+
+export default (app: App) => {
+  app.use(i18nPlugin);
+};
+```
+
+### jsx
+
+`jsx: true`를 사용하여 Vue JSX를 사용할 수 있습니다.
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:163]:
```suggestion
`jsx: true`를 설정하여 Vue JSX를 사용할 수 있습니다.
```

File: src/content/docs/ko/guides/integrations-guide/vue.mdx [hunk 15]
```diff
@@ -0,0 +1,192 @@
...
+```js title="astro.config.mjs"
+import { defineConfig } from 'astro/config';
+import vue from '@astrojs/vue';
+
+export default defineConfig({
+  // ...
+  integrations: [
+    vue({
+      jsx: {
+        // ion- 으로 시작하는 것과 같은 모든 커스텀 요소를 처리합니다
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/integrations-guide/vue.mdx:186]:
```suggestion
        // ion- 으로 시작하는 모든 태그를 맞춤 요소로 처리합니다.
```

**Review by jsparkdev — COMMENTED** [review]
Thank you for your hard work!

If you have any questions, please leave them.

File: src/content/docs/ko/guides/backend/supabase.mdx [hunk 16]
```diff
@@ -0,0 +1,501 @@
+---
+title: Supabase & Astro
+description: 'Supabase 를 이용하여 당신의 프로젝트에 백앤드자원을 추가합니다'
+type: backend
+service: Supabase
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+
+[Supabase](https://supabase.com/) 는 오픈소스로, Firebase의 상위호환 입니다. Authentication(인증), Postgres Database(DB), Edge Function(엣지 함수), Realtime(실시간 데이터 fetch) 및 Storage(스토리지) 를 제공합니다.
```
[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/ko/guides/backend/supabase.mdx:10]: 아~~! Supabase에서 제공하는 언어가 영어뿐이라, 한글만 기재하면 이용이 어려울 것으로 보여져서 영어뒤에 번역을 넣은 내용이었습니다~~!

File: src/content/docs/ko/guides/backend/supabase.mdx [hunk 17]
```diff
@@ -0,0 +1,501 @@
...
+
+## Astro 에서 Supabase 초기화 하기
+
+### 선행조건
+
+- 만일 Supabase 프로젝트가 없을 경우, [supabase.com](https://supabase.com/)에서 무료로 가입하고, 새로운 프로젝트를 생성할 수 있습니다.
```
[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/ko/guides/backend/supabase.mdx:16]: 감사합니다!!!

File: src/content/docs/ko/guides/backend/supabase.mdx [hunk 18]
```diff
@@ -0,0 +1,501 @@
...
+- [server-side rendering](/ko/guides/server-side-rendering/)이 활성화 되어있는 Astro 프로젝트여야 합니다.
+- Supabase 프로젝트의 자격증명을 확인해야 합니다. Supabase 프로젝트의 **설정 > API** 탭에서 아래의 두개의 자격증명을 확인이 가능합니다.
+  - `SUPABASE_URL`: Supabase 프로젝트의 URL 입니다.
+  - `SUPABASE_ANON_KEY`: Supabase 프로젝트의 익명키 입니다.
+
+### Supabase 자격증명 추가하기
+
+Astro 프로젝트에 Supabase의 자격증명을 추가하기 위해, `.env` 파일을 아래와 같이 추가합니다.
```
[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/ko/guides/backend/supabase.mdx:24]: 전체적으로 '~다' 로 번역 되어있는 것으로 보여져서 '~다' 로 수정처리 하겠습니다!

File: src/content/docs/ko/guides/backend/supabase.mdx [hunk 19]
```diff
@@ -0,0 +1,501 @@
...
+  readonly SUPABASE_URL: string
+  readonly SUPABASE_ANON_KEY: string
+}
+
+interface ImportMeta {
+  readonly env: ImportMetaEnv
+}
+```
+
+:::팁
```
[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/ko/guides/backend/supabase.mdx:46]: 감사합니다!!! :)

File: src/content/docs/ko/guides/backend/supabase.mdx [hunk 20]
```diff
@@ -0,0 +1,501 @@
...
+    - **supabase.ts**
+  - env.d.ts
+- .env
+- astro.config.mjs
+- package.json
+</FileTree>
+
+## Supabase로 인증 추가하기
+
+Supabase는 즉시 인증기능을 제공해줍니다. 이메일/비밀번호 인증뿐만 아니라, GitHub, Google, Kakao 와 같은 기타 여러 제공업체의 인증방식과, Oauth 인증을 지원합니다.
```
[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/ko/guides/backend/supabase.mdx:109]: Kakao를 실제로 제공해주고 있어서, 추가해두었습니다~~

[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/ko/guides/backend/supabase.mdx:109]: 또한,

[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/ko/guides/backend/supabase.mdx:109]: Supabase에서 이메일,비밀번호로 인증하는 방식 및 그 밖의 인증 제공업체들의 OAuth 인증방식을 통해 인증기능을 제공합니다

File: src/content/docs/ko/guides/backend/supabase.mdx [hunk 21]
```diff
@@ -0,0 +1,501 @@
...
+
+### 인증페이지 만들기
+
+이제 서버 엔드포인트를 만들었으니, 이를 사용할 페이지를 생성합니다.
+
+- `src/pages/register`: 새로운 사용자를 등록할 양식이 포함되어 있습니다.
+- `src/pages/signin`: 사용자의 로그인양식이 포함되어 있습니다.
+- `src/pages/dashboard`: 인증된 사용자만 접근가능한 페이지가 포함되어 있습니다.
+
+`src/pages` 폴더에 위와 같은 페이지를 만듭니다. 이제 프로젝트에 아래와 같은 파일이 포함되어야 합니다:
```
[the remark this answers is not in the document]
Comment (yeonjulee1005) [reply src/content/docs/ko/guides/backend/supabase.mdx:225]: 영문에도 콜론 ':' 으로 종료가 되어있어서 그대로 남겨 두었습니다~~ 위의 내용들에도 동일하게 반영해 두었습니다~!

**Review by jsparkdev — APPROVED** [review]
고생 많으셨습니다!

**Review by dreyfus92 — APPROVED** [review]
Thank you for handling this @yeonjulee1005. Also big shoutout to @jsparkdev for reviewing this PR 😁

Welcome to `astro-docs` @yeonjulee1005 🥳

**Pull request merged**